anki/collection.py

```diff
@@ -77,14 +77,33 @@ def timezoneOffset() -> int:
 class _Collection:
     db: Optional[DB]
     sched: Union[V1Scheduler, V2Scheduler]
+    # Timestamp when the collection was created in seconds since the epoch.
     crt: int
+    # Last modification time of the collection in milliseconds since the epoch.
     mod: int
+    # Last modification time of the schema in milliseconds since the epoch.
     scm: int
-    dty: bool  # no longer used
+    # No longer used.
+    dty: bool
+    # TODO: document
     _usn: int
+    # Last modification time of the schema as of the last full upload.
     ls: int
+    # Various configuration, stored in the database as a JSON string.
     conf: Dict[str, Any]
-    _undo: List[Any]
+    # Undo data. If None, there is nothing to undo.
+    # Otherwise, list of [type: int, undoName: str, ... optionally more ...].
```

Comment:
Might be nicer to move this longer explanation to the part of the file that deals with undoing.